x-pack/legacy/plugins/siem/server/lib/ip_details/mock.ts

```diff
@@ -252,37 +252,6 @@ export const formattedEmptySource = {
   },
 };
 
-export const mockDomainsResponseBuckets: DomainsBuckets[] = [
-  {
-    key: 'example.com',
-    uniqueIpCount: {
-      value: 805,
-    },
-    lastSeen: {
-      value: 1554920919000,
-      value_as_string: '2019-04-10T18:28:39.000Z',
-    },
-    bytes: {
-      value: 974964465,
-    },
-    packets: {
-      value: 16946245,
-    },
-    direction: {
-      buckets: [
-        {
-          key: NetworkDirectionEcs.outbound,
-          doc_count: 51668,
-        },
-        {
-          key: NetworkDirectionEcs.inbound,
-          doc_count: 25681,
-        },
-      ],
-    },
-  },
-];
-
 export const mockFormattedSource: DomainsEdges[] = [
```

Comment:
This mock is unused. Doing a global search for `DomainsEdges` will show a few other touch points to clean up as part of removing the Domains table. Wiping out `domainsSchema` from `/server/graphql/ip_details/schema.gql.ts` and running the gql generator should expose most of them.